18_regex/slides.md

```diff
@@ -936,14 +941,13 @@ If you are using the same regex many times consider compiling it once and cachin
 
 ```scala
 // Recompiles the regex on every iteration
-strings.filter(_.matches("boban.+jones"))
+strings.findFirstIn(_.matches("boban.+jones"))
 
 // Compile it once
 val compiled = "boban.+jones".r
-strings.collect { case s@compiled(_*) => s }
+...
 ```
 
-(Example is a little awkward,
 see also [SO post](https://stackoverflow.com/questions/3021813/how-to-check-whether-a-string-fully-matches-a-regex-in-scala))
```

Comment:
This link doesn't really make sense now. It was part of the line above.

I think just delete it.

Reply:
removedz